tests/conftest.py

```diff
@@ -7,22 +8,16 @@
 collect_ignore = ["test_graphql.py"] if sys.version_info >= (3, 10) else []
 
 
-@pytest.fixture(
-    params=[
-        pytest.param(
-            {"backend": "asyncio", "backend_options": {"use_uvloop": False}},
-            id="asyncio",
-        ),
-        pytest.param({"backend": "trio", "backend_options": {}}, id="trio"),
-    ],
-    autouse=True,
-)
-def anyio_backend(request, monkeypatch):
-    monkeypatch.setattr(TestClient, "async_backend", request.param)
-    return request.param["backend"]
+@pytest.fixture
+def no_trio_support(anyio_backend_name):
+    if anyio_backend_name == "trio":
+        pytest.skip("Trio not supported (yet!)")
```

Comment:
```suggestion
        pytest.skip("Trio not supported (yet!)")

@pytest.fixture(params=["asyncio", "trio"])
def anyio_backend(request):
    return request.param
```

@uSpike I could add this back in?